frontend/src/app/data-entry/location-form/location-regions-form/location-regions-form.component.html

```diff
@@ -0,0 +1,12 @@
+<form>
+    <p class="form-text">
+        Pick one or more regions that contain the location represented here.
```

Comment:
Maybe this can be a bit more clear that this is (at least for now) about regions that are actually mentioned.

E.g. "Select regions by which the text identifies or contextualises the location."

Same comment for settlements and structures.